### PR TITLE
Tisane Labs - New Components

### LINKED ISSUE #6122: [Components] Tisane Labs
### App
Tisane Labs
https://tisane.ai/

Triggers:
1. Text Analyzed: Triggered when a text is analyzed for language, entities, sentiment, etc.
2. Abuse Detected: Triggered when harmful or offensive content is identified in a text.

Actions:
1. Analyze Text: Analyze text for language, entities, sentiment, and other insights.
2. Detect Abuse: Identify harmful or offensive content in text.
3. Translate Text: Translate text between supported languages.

_This list of suggested components was generated using ChatGPT_

### PR DESCRIPTION
**Pull request by michelle0927**
Resolves #6122 

**Actions**

- Analyze Text
- Translate Text
- Detect Language
- Generate Image From Text

### CODE REVIEW

File: components/tisane_labs/actions/detect-language/detect-language.mjs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      label: "Delimiter",
+      description: "A regular expression to segment the fragment; by default, languages are detected globally",
+      optional: true,
+    },
+  },
+  async run({ $ }) {
+    const response = await this.tisaneLabs.detectLanguage({
+      data: {
+        content: this.content,
+        languages: this?.languages?.length
```
Comment (alysonturing) [inline components/tisane_labs/actions/detect-language/detect-language.mjs:37]: I suggest doing like this: `this.languages?.join(",") || ""`

File: components/tisane_labs/common/constants.mjs
Comment (alysonturing) [inline components/tisane_labs/common/constants.mjs:1]: You could add a label in these arrays for a better user experience

**Review by alysonturing — DISMISSED** [review]
LGTM! Thank you for your valuable contribution!
I added a couple of suggestions for improvement

**Review by andrewjschuang — COMMENTED** [review]
`detect-language` action has passed:
Just at observation, it detected the greek language even though it's not on the language list I provided. But I don't think that's an issue with the component anyway.

image
image

**Review by andrewjschuang — COMMENTED** [review]
`analyze text` action passed:

Required props:
image
image

All props:
image
image
image
image
image

**Review by andrewjschuang — COMMENTED** [review]
`translate-text` action passed:

image
image

**Pull request merged**